### ISSUE TITLE
Lut 3D module has disappeared

### ISSUE THREAD

**Issue by jmviade** [darktable-org/darktable#11770]
Hi, 

Yesterday I was trying to use the Lut 3D module but it doesn't appear in the list of modules.  I have defined the folder for lut3d files in Preferences > Processing  and I've restarted darktable however this module doesn't appear.

I've configured the view "all modules" but even in this case the "lut 3D" is missing.

I'm working under Windows 10 with Darktable 3.8.1 principally with raw .CR2  files from Canon cameras: 450D and 760D

Thanks in advance for any suggestion or comment.

Joan Miquel Viadé

**Comment by phil20099** [comment]
Quick question : what happens if you try to search for "lut" using the search box towards the top right of the darkroom ?


![Screenshot-20220509063949-337x230](https://user-images.githubusercontent.com/60315272/167347242-991e827a-b5fe-4636-ad1c-faca1ac0969c.png)

**Comment by kmilos** [comment]
Do you see the unable to load lut3d.dll message in the log? Could be similar to https://github.com/darktable-org/darktable/issues/9389 or https://github.com/darktable-org/darktable/issues/10007 ...

**Comment by jmviade** [comment]
Excuse my question but where can I find the .log file? May I activate previously some option?
thanks in advance.

**Comment by kmilos** [comment]
Yep, that's the message. You might want to ask on the other two topics if anyone found a solution....

> [iop_load_module] failed to open `lut3d': 'C:\Program Files\darktable\lib\darktable/plugins\liblut3d.dll': No se encontró el proceso especificado.

Yep, that's the message. As this is probably a duplicate, you might want to continue the discussion on the other two links to see if someone found a solution.

**Comment by jmviade** [comment]
Hi, kmilos.  

I was trying most of the solutions and tricks that are explained in the two threads that you suggested to me but the "Lut 3D" module doesn't appear either.

For example I installed msys2 program a through its 'pacman' tool also the gmic and openvc modules, I also checked for other dependencies, etc.... however it doesn't working properly.

Thanks, but I don't know what I can do in a windows 10 PRO environment. Any other suggestions.

**Comment by kmilos** [comment]
Please continue the conversation on the other two issues, it is more likely someone will reply there.
